integration-tests/tests/import-cohort-data.integration-test.js

```diff
@@ -1,26 +1,25 @@
-const dataExplorer = require('../utils/data-explorer-utils')
 const { testUrl } = require('../utils/integration-config')
 const { withWorkspace } = require('../utils/integration-helpers')
-const { findInGrid, exactClick, click, findIframe, signIntoTerra, select } = require('../utils/integration-utils')
+const { findInGrid, click, clickable, fillIn, findIframe, input, signIntoTerra, select, svgText } = require('../utils/integration-utils')
 
 
 const cohortName = `terra-ui-test-cohort`
 
 test('import cohort data', withWorkspace(async ({ workspaceName }) => {
   await page.goto(testUrl)
   await signIntoTerra(page)
-  await click(page, 'Browse Data')
-  await click(page, '1000 Genomes Low Coverage')
+  await click(page, clickable({ textContains: 'Browse Data' }))
+  await click(page, clickable({ textContains: '1000 Genomes Low Coverage' }))
 
   const frame = await findIframe(page)
-  await dataExplorer.clickTextInAnyNS(frame, 'Has WGS Low')
-  await click(frame, 'Save cohort')
-  await dataExplorer.fillIn(frame, 'name', cohortName)
-  await dataExplorer.click(frame, 'Save')
+  await click(frame, svgText({ textContains: 'Has WGS Low Cov' }))
+  await click(frame, clickable({ textContains: 'Save cohort' }))
+  await fillIn(frame, input({ placeholder: 'cohort name' }), cohortName)
```

Comment:
TOL: Ouch! Scary that we have to rely on placeholder to find this input. 👍 to making that reliance explicit and clearly visible in the test code!